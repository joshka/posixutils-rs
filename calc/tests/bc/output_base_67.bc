obase = 67
0
1
1240
128.906662
quit
