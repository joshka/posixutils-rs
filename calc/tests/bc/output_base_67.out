0
 01
 18 34
 01 61.60 50 00 25
